images/base/files/usr/local/bin/entrypoint

```diff
@@ -567,6 +567,7 @@ fix_product_name
 fix_product_uuid
 select_iptables
 enable_network_magic
+update-ca-certificates
```

Comment:
how much time does this take when it isn't mounted?

we're generally pretty sensitive to startup time and we might be able to do something more clever if this is slow

it's also unnecessary when the container is restarted, right? (unless certs were updated in between ...)